addons/godot-next/resources/behavior.gd

```diff
@@ -0,0 +1,39 @@
+extends Resource
+class_name Behavior
+
+##### CLASSES #####
+
+##### SIGNALS #####
+
+##### CONSTANTS #####
+
+##### EXPORTS #####
+
+##### PROPERTIES #####
+
+var owner = null
+var enabled: bool = false
+
+##### NOTIFICATIONS #####
+
+##### OVERRIDES #####
+
+##### VIRTUALS #####
```

Comment:
We should probably add, for auto-completion's sake, `pass`-ing virtual methods for each of the notifications.